README.md

```diff
@@ -36,9 +36,13 @@ set([1, 2, 3]).union([3, 4, 5])           // [1, 2, 3, 4, 5]
 set([1, 2, 3]).intersection([4, 5, 6])    // []
 set([1, 2, 3]).intersection([3, 4, 5])    // [3]
 
-// difference
+// difference:
 
 set([1, 2, 3]).minus([1, 3])              // [2]
+
+// remove duplicates
```

Comment:
I think "remove duplicates" is an _effect_, not the definition of what's actually happening.  That is, this is how you _create a set_, the definitional _effect_ of which is to remove duplicates.  I would just say `// create an arrayset instance`